### DISCUSSION TITLE
the ACTUAL green/blue shots...

### DISCUSSION THREAD

**Discussion by DaveFan19** [Pico8430/Indie-Cross-PE-Port#19] [Show and tell]
its fixed correctly now!

https://user-images.githubusercontent.com/99294235/173251658-5bf154c1-77c6-4e8f-b37a-35f953673f26.mp4

**Comment by jmxamongusmodder** [comment]
> its fixed correctly now!
> 
>  2022-06-12.17-08-50.mp4
i would really need this (i could add it to my pr?)

**Reply by DaveFan19** [reply]
uhh sure i guess

> > its fixed correctly now!
> > 2022-06-12.17-08-50.mp4
> > i would really need this (i could add it to my pr?)

uh sure i guess

**Reply by DaveFan19** [reply]
> uhh sure i guess
> 
> > > its fixed correctly now!
> > > 2022-06-12.17-08-50.mp4
> > > i would really need this (i could add it to my pr?)
> 

actually i cant post it here, github doesn't allow big files
